apps/readest-app/src/app/reader/page.tsx

```diff
@@ -28,5 +28,9 @@ export default function Page() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [appService?.hasUpdater, settings.autoCheckUpdates]);
 
-  return <Reader />;
```

Comment:
Please revert back to the original state if the code changes are not intentional.